README.md

```diff
@@ -1,7 +1,163 @@
 # scradd
+![](https://cdn.discordapp.com/icons/938438560925761619/d886f0b0867e67c2211b7086c0651590.webp?size=80)
+## About
 
-Bot for Scratch Addons Discord server
+Scradd is the Discord bot for the [Scratch Addons Discord server](https://discord.gg/Cs25kzs889). It was partially made because servers' one year anniversary, and partially because we wanted a bot that we had ultimate customization over.
 
-[Cloud hosted on opeNode.io](https://www.openode.io/)
+Logo made by Weirdo#8115 (Discord) and @retronbv.
 
-todo write something here because that's probably why this file was created, but maybe not idk, I should google who created the readme file and the purpose for it. that's something I don't know. ok cool, thanks Wikipedia <https://en.wikipedia.org/wiki/README#History> now I have more knowledge, but also I have wasted 5m of my life typing this which is probably bad...
+Cloud-hosted on [opeNode.io](https://www.openode.io/).
+
+## Table of Contents
+ [Features](#features) | [Contributing](#contributing) 
+| --- | ---
+| [Potatoboard](#potatoboard) | [How to contribute](#how-to-contribute)
+| [Suggestions - /suggestion](#suggestions) | [Contributors](#contributors)
+| [Bug Reports - /bugreport](#bug-reports) |
+| [Addon Info - /addon](#addon-addon) |
+
+
+
+## Features
+
+### Potatoboard
+  
+Messages with more than 8 :potato: reactions are shown in the #potatoboard channel. 
+
+Potatoboard can be used to higlight messages that the community finds noteworthy or funny: like a good drawing, a message funny out of context or a good meme in #memes.
+
+
+![](https://cdn.discordapp.com/attachments/901225174974726177/939015132720287784/unknown.png) 
+
+
+#### Related: /explorepotatos [minimum-reactions] [user] [channel]
+
+Gets a random message from the potatoboard. 
+
+
+| Argument | Description | Optional?|
+| --- | ---| --- |
+| minimum-reactions | Minimum reactions the message needs to have | yes|
+| user | The author of the message | yes|
+| channel | The channel that the message was originally in | yes |
+
+---
+
+### React to messages including certain strings
+
+Scradd reacts to messages that include certain strings. 
+
+What strings? Well... find out yourself! ![](https://cdn.discordapp.com/emojis/902948518002573364.webp?size=22&quality=lossless)
+
+---
+
+### Suggestions
+
+Create and manage suggestions in the #suggestions channel.
+
+**Subcommands:** create | edit | answer | delete
```

Comment:
they can scroll down to find this

Reply:
ik but it's better in one line to introduce imo